README.md

```diff
@@ -54,3 +54,21 @@ python run_processors.py [--aliases] [--sleep_time]
 ```
 - `aliases` - aliases list to run (aliases of available processors:`enrollment`, `video_views`, `discussion`, `student_step`, `course_activity`)
 - `sleep_time` - log directory rescan period (seconds, default: 5 minutes).
+
+
+## Unit tests
+Instructions to run tests manually go below.
+
+##### To run unit tests manually, follow the next steps:
+* Ensure to place the source code in one of the edx-platform directories.
```

Comment:
I do not understand what it means. Do I copy a test file into lms? Do I clone the repo somewhere? Where to exactly? Could you maybe give an example folder structure here?

Reply:
Updated README repectively